### PR TITLE
Fix Compiler Warnings (new rustc)

### PR DESCRIPTION
**Pull request by shawntabrizi**
This simply fixes some compiler warnings that arose from the latest rustc

### CODE REVIEW

File: primitives/core/benches/bench.rs [hunk 1]
```diff
@@ -49,87 +46,133 @@ fn bench_twox_128(b: &mut Bencher, key: &Vec<u8>) {
...
+			|b| b.iter(|| sp_core::ed25519::Pair::verify(&sig, &msg, &public)),
+		);
+	}
+
+	group.finish();
 }
 
 fn bench_sr25519(c: &mut Criterion) {
-	c.bench_function_over_inputs("signing - sr25519", |b, &msg_size| {
+	let mut group = c.benchmark_group("ed25519");
```
Comment (shawntabrizi) [inline primitives/core/benches/bench.rs:111]:
```suggestion
	let mut group = c.benchmark_group("sr25519");
```

File: primitives/core/benches/bench.rs [hunk 2]
```diff
@@ -49,87 +46,133 @@ fn bench_twox_128(b: &mut Bencher, key: &Vec<u8>) {
...
+			|b| b.iter(|| sp_core::sr25519::Pair::verify(&sig, &msg, &public)),
+		);
+	}
+
+	group.finish();
 }
 
 fn bench_ecdsa(c: &mut Criterion) {
-	c.bench_function_over_inputs("signing - ecdsa", |b, &msg_size| {
+	let mut group = c.benchmark_group("ed25519");
```
Comment (shawntabrizi) [inline primitives/core/benches/bench.rs:141]:
```suggestion
	let mut group = c.benchmark_group("ecdsa");
```

**Review by bkchr — APPROVED** [review]

**Review by adoerr — APPROVED** [review]

File: bin/node/executor/benches/bench.rs [hunk 3]
```diff
@@ -139,15 +139,14 @@ fn construct_block<E: Externalities>(
...
-
 fn test_blocks(genesis_config: &GenesisConfig, executor: &NativeExecutor<Executor>)
 	-> Vec<(Vec<u8>, Hash)>
 {
 	let mut test_ext = new_test_ext(genesis_config);
 	let mut block1_extrinsics = vec![
 		CheckedExtrinsic {
 			signed: None,
-			function: Call::Timestamp(pallet_timestamp::Call::set(42 * 1000)),
+			function: Call::Timestamp(pallet_timestamp::Call::set(0)),
```
[the remark this answers is not in the document]
Comment (shawntabrizi) [reply bin/node/executor/benches/bench.rs:149]: No it was not. This fixes it, but dunno how correct it is otherwise

**Comment by shawntabrizi** [comment]
> error: internal compiler error: cannot lex `source_file` without source: /builds/parity/substrate/target/debug/build/substrate-test-runtime-76dba39500304725/out/wasm_binary.rs
thread 'rustc' panicked at 'Box<Any>', /rustc/126561cb31e8ebe1e2dd9dfd0d3ca621308dc56f/library/std/src/panic.rs:59:5

??

**Pull request merged**